tests/framework/PostProcessors/MCSimporterPostProcessor/test_MCSimporter.xml

```diff
@@ -1,11 +1,11 @@
 <Simulation verbosity="debug">
   <TestInfo>
-    <name>framework/PostProcessors/MCSimporterPostProcessor.MC_importer_Saphire</name>
+    <name>framework/PostProcessors/MCSimporterPostProcessor.MC_importer</name>
```

Comment:
MC_importer should be changed to __MCSimporter__. The name should be consistent with the 'tests' file.

Reply:
Fixed!